taipy/gui/_renderers/factory.py

```diff
@@ -169,6 +169,7 @@ class _Factory:
                 ("on_change", PropertyType.function),
                 ("format",),
                 ("width", PropertyType.string_or_number),
+                ("disable_date_config",PropertyType.dynamic_dict)
```

Comment:
SHould this be dynamic ?

Reply:
I believe it should be dynamic, as some keys in the dictionary are optional. Please correct me if I'm wrong.

Reply:
> I believe it should be dynamic, as some keys in the dictionary are optional. Please correct me if I'm wrong.

Dynamic means that the bound variable can be updated.

Reply:
Thank you for the clarification, @FredLL-Avaiga. I will change it to dict.

I have fixed the checks that failed due to spelling. The remaining tests are failing because no test cases were written, correct?